### DISCUSSION TITLE
Unable to authenticate to MongoDB

### DISCUSSION THREAD

**Discussion by UromPLAt** [dbeaver/dbeaver#19468] [General]
Hello peeps, I'm attempting to connect to a remote MongoDB via an SSH Tunnel, the SSH connection is successful but authenticating to the DB server is not.

I'm using the same connection details in both NoSQLBooster and in the DBeaver Ultimate, and in NoSQLBooster it works and in DBeaver it does not.
The connection URL is: mongodb://localhost:27017/<database-name>
Authentication: MongoDB authentication
Mechanism: NONE

In NoSQLBooster it doesn't prompt me for a username/password and in DBeaver it does.

In DBeaver I get this error:

> com.mongodb.MongoTimeoutException: Timed out after 5000 ms while waiting to connect. Client view of cluster state is {type=UNKNOWN, servers=[{address=localhost:27017, type=UNKNOWN, state=CONNECTING, exception={com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused: connect}}]

[NoSQLBooster log](https://pastebin.com/uU4Bt9uB)
[DBeaver log](https://pastebin.com/7mXPBP4J)

Any idea what I'm doing wrong?

**Comment by serge-rider** [comment]
Hi,

In order to use SSH tunnel you need to use host/port configuration (i.e. you cannot use URL-based connections).
Could you send more information about how your connection is configured? Some screenshorts would be very helpful Please remove any sensitive data before posting.

(Or you can create a ticket on dbeaver.com)
